rim/tests.py

```diff
@@ -1,3 +1,39 @@
-from django.test import TestCase
+from django.test import TestCase, Client
 
 # Create your tests here.
+class TestHttpResponse(TestCase):
+    # Url dictionary to hold names for existing rim web pages, add as neccessary 
+    urlDictionary = {
+        "Admin Panel": '/admin/',
```

Comment:
It's not the normal practice to unit test third-party code. Normally, you would trust your upstream dependencies to work correctly. Since Django does all the work to make (and test) the admin panel, we shouldn't need to test it ourselves. With that said, we do _kind of_ control it with rim/admin.py which could possibly contain errors... but let's just leave that for later.